pkg/services/serviceaccounts/database/store.go

```diff
@@ -530,3 +530,18 @@ func (s *ServiceAccountsStoreImpl) RevertApiKey(ctx context.Context, saId int64,
 	}
 	return nil
 }
+
+func userDeletions(dialect migrator.Dialect) []string {
```

Comment:
nit: if this function is only called from the service account store, might make sense to rename it to `serviceAccountDeletions` or something like that.